src/elements/ScrollContainer/ScrollContainer.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { Text, View } from 'react-native';
+import { SpacerInline } from '@elements/SpacerInline';
+import EndOfScrollWrapper from './EndOfScrollWrapper';
+import Checkbox from '../Checkbox/Checkbox';
+import styles from './ScrollContainer.styles';
+
+interface ScrollContainerProps {
+	onScrollToEnd?: Function;
+	onPress?: Function;
+	setChecked: Function;
+	documentText: string;
+	checked: boolean;
```

Comment:
Should we be giving `Checkbox` logic to the `ScrollContainer`?

I feel like it would make more sense to set up these components on the `TermsScreen.tsx` like:

```
<ScrollContainer   {...props}>
    <CheckBox 
        {...props}
    />
</ScrollContainer>
```

That way we can reuse the `ScrollContainer` in other parts of the App without having to remember whether or not to pass a bunch of `Checkbox` props down through the `ScrollContainer`

My understanding of this could be wrong though, please let me know your thoughts! 👍